damnit/backend/extraction_control.py

```diff
@@ -85,12 +85,14 @@ class ExtractionRequest:
     match: tuple = ()
     mock: bool = False
     update_vars: bool = True
+    data_location: str = 'localhost'
```

Comment:
I see what you're doing, but I think either the name `data_location` or the default 'localhost' is rather confusing. The data location is always on a network filesystem in any case, the distinction is whether it's persistently mounted at a standard path or whether we mount it temporarily at an unpredictable one.

Reply:
yea, I did not think too hard about names (I thought this would disappear soon ™️  anyway). I'm open for suggestions, as I'm not sure what would make it clearer. `mount='default'`?

Reply:
Yeah, `mount` or `mount_host` might be clearer. I might also make the default `None` to make it clear that this is actually different behaviour rather than just a different target.